types/src/api/plugins/participation/types/participation.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
I think I would flatten a bit and have a single types.rs file instead of a module